core/src/main/java/io/github/fourlastor/game/demo/state/unit/Mon.kt

```diff
@@ -38,12 +39,24 @@ class Mon(
 
     fun onRoundStart(): Action = stacks.onRoundStart(this)
 
+    fun onTurnStart(): Action = stacks.onTurnStart(this)
+
     fun onRoundEnd() {
         stacks.tickStacks()
     }
 
     fun addEffect(effect: Effect) {
-        stacks.addStack(effect, 3)
+        addEffect(effect, 3)
+    }
+
+    fun addEffect(effect: Effect, numberStacks: Int) {
+        stacks.addStack(effect, numberStacks)
+    }
+
+    fun getEffects(): ObjectIntMap<Effect> = stacks.getEffects()
```

Comment:
change this to `: Map<Effect>`

Reply:
It complains about a type mismatch

Reply:
Ah right this is a special type, nevermind